vernac/command.ml

```diff
@@ -549,13 +549,12 @@ let interp_mutual_inductive (paramsl,indl) notations cum poly prv finite =
     lift_implicits (Context.Rel.nhyps ctx_params) impls) arities in
   let arities = List.map pi1 arities and aritypoly = List.map pi2 arities in
   let impls = compute_internalization_env env0 ~impls (Inductive (params,true)) indnames fullarities indimpls in
-  let implsforntn = compute_internalization_env env0 Variable indnames fullarities indimpls in
```

Comment:
I was trying to review the code, esp the fix, but most of this patch is just threading the env (that is a very welcome change).  Am I right the fix to the bug is literally this line being removed?

Reply:
@herbelin any comment?